internal/apis/dbaas/implementation/client.go

```diff
@@ -183,8 +183,8 @@ func (client *Client) PutConfiguration(publicCloudId int64, publicCloudProjectId
 	return result.Data, nil
 }
 
-func (client *Client) GetConfiguration(publicCloudId int64, publicCloudProjectId int64, dbaasId int64) (map[string]string, error) {
-	var result helpers.NormalizedApiResponse[dbaas.StringMap]
+func (client *Client) GetConfiguration(publicCloudId int64, publicCloudProjectId int64, dbaasId int64) (map[string]any, error) {
+	var result helpers.NormalizedApiResponse[map[string]any]
```

Comment:
I see we've deleted usage of `dbaas.StringMap` but we haven't actually deleted the type, is there a reason for this ?